velox/functions/prestosql/window/WindowFunctionsRegistration.h

```diff
@@ -17,6 +17,17 @@
 
 namespace facebook::velox::window {
 
-void registerWindowFunctions();
+namespace prestosql {
+
+void registerAllWindowFunctions();
+
+} // namespace prestosql
+
+#ifdef VELOX_ENABLE_BACKWARD_COMPATIBILITY
+/// TODO Remove after updating Prestissimo.
+void registerAllWindowFunctions() {
```

Comment:
The original function name was registerWindowFunctions.

To make is easier to delete this code, let's organize it like so:

```
namespace facebook::velox::window::prestosql {
void registerAllWindowFunctions();
}

#ifdef VELOX_ENABLE_BACKWARD_COMPATIBILITY
namespace facebook::velox::window{
void registerWindowFunctions() {
   prestosql::registerAllWindowFunctions();
}
}
#endif
```

Reply:
Sorry my bad. I missed that.